### PR TITLE
Feature/ikdt 674 origin module

### PR DESCRIPTION
**Pull request by aks8m**
Initial working origin packaging implementation.

### CODE REVIEW

File: mvnw [hunk 1]
```diff
@@ -0,0 +1,259 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
```
Comment (stenkarl) [inline mvnw:2]: Do we use mvnw? I thought it was an older wrapper script we no longer use.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        <origin.working.directory>${project.build.directory}/origin-sources</origin.working.directory>
+        <source.zip>snomedct-us-20240901T120000Z.zip</source.zip>
+        <packager.name>IKM Dev</packager.name>
+        <origin.url>https://www.nlm.nih.gov/healthit/snomedct/us_edition.html</origin.url>
+        <origin.namespace>3094dbd1-60cf-44a6-92e3-0bb32ca4d3de</origin.namespace>
+
+        <!-- starter data -->
+        <composer.version>1.5.0</composer.version>
+        <tinkar-starter-data.version>1.0.0</tinkar-starter-data.version>
+        <tinkar-core.version>1.72.0</tinkar-core.version>
```
Comment (stenkarl) [inline pom.xml]: You could update this to 1.76.0 if you'd like.

**Review by stenkarl — APPROVED** [review]

File: mvnw (hunk 1 above)
Comment (aks8m) [reply mvnw:2]: tinkar-core has these in every maven module. I'm not a fan but was trying to be consistent.

**Review by rbradley813 — APPROVED** [review]
Look good!

**Pull request merged**